src/site/twitch.tv/modules/settings/SettingsModule.vue

```diff
@@ -20,6 +21,17 @@ const { markAsReady } = declareModule("settings", {
 
 const ctx = useSettingsMenu();
 
+// hook for the top nav bar
+const topnav = useComponentHook<Twitch.TopNavBarComponent>(
+	{
+		predicate: (n) => n.props && n.props.isWhispersLoaded && n.props.languageCode,
```

Comment:
This predicate should have a class selector

Reply:
I believe the parent is the root, so I am not sure if specifying a selector would make a difference here